File: testingFiles/testSourceCode/MainTreeSet.java
import java.util.*;

public class MainTreeSet {
    public static void main(String args[]) {
        
        int N = Integer.parseInt(args[0]);
        int tot = N / 10;

        Random random = new Random();
        TreeSet<Integer> tree = new TreeSet<Integer>();
        for (int te = 0; te < 10; ++te) {
            for (int i = 1; i <= tot; ++i) tree.add(random.nextInt());
        }

    }
}

File: testingFiles/testSourceCode/MainTreeSet.cpp
#include "Utility.h"
#include "ArrayList.h"
#include "LinkedList.h"
#include "HashSet.h"
#include "HashMap.h"
#include "TreeSet.h"
#include "TreeMap.h"

#include<cstdlib>

class Hashint {
    public:
         static int hashcode(int obj) {
             return obj;
         }
};

int main(int argc, char **argv) {

    int N = atoi(argv[1]);
    int tot = N / 10;

    srand(time(NULL));
    TreeSet<int> tree;
    for (int te = 0; te < 10; ++te) {
        for (int i = 1; i <= tot; ++i) tree.add(rand());
    }

    return 0;

}
